addons/hr_expense/report/hr_expense_report.py

```diff
@@ -1,64 +1,44 @@
-        'date': fields.date('Date ', readonly=True),
-        'create_date': fields.datetime('Creation Date', readonly=True),
-        'product_id':fields.many2one('product.product', 'Product', readonly=True),
-        'journal_id': fields.many2one('account.journal', 'Force Journal', readonly=True),
-        'product_qty':fields.float('Product Quantity', readonly=True),
-        'employee_id': fields.many2one('hr.employee', "Employee's Name", readonly=True),
-        'date_confirm': fields.date('Confirmation Date', readonly=True),
-        'date_valid': fields.date('Validation Date', readonly=True),
-        'department_id':fields.many2one('hr.department','Department', readonly=True),
-        'company_id':fields.many2one('res.company', 'Company', readonly=True),
-        'user_id':fields.many2one('res.users', 'Validation User', readonly=True),
-        'currency_id': fields.many2one('res.currency', 'Currency', readonly=True),
-        'price_total':fields.float('Total Price', readonly=True, digits_compute=dp.get_precision('Account')),
-        'delay_valid':fields.float('Delay to Valid', readonly=True),
-        'delay_confirm':fields.float('Delay to Confirm', readonly=True),
-        'analytic_account': fields.many2one('account.analytic.account','Analytic account',readonly=True),
-        'price_average':fields.float('Average Price', readonly=True, digits_compute=dp.get_precision('Account')),
-        'nbr':fields.integer('# of Lines', readonly=True),  # TDE FIXME master: rename into nbr_lines
-        'no_of_products':fields.integer('# of Products', readonly=True),
-        'no_of_account':fields.integer('# of Accounts', readonly=True),
-        'state': fields.selection([
-            ('draft', 'Draft'),
-            ('confirm', 'Waiting confirmation'),
-            ('accepted', 'Accepted'),
-            ('done', 'Done'),
-            ('cancelled', 'Cancelled')],
-            'Status', readonly=True),
-    }
+
+    date = fields.Date(readonly=True)
+    create_date = fields.Datetime(string='Creation Date', readonly=True)
+    product_id = fields.Many2one('product.product', string='Product', readonly=True)
+    journal_id = fields.Many2one('account.journal', string='Force Journal', readonly=True)
+    product_qty = fields.Float(string='Product Quantity', readonly=True)
+    employee_id = fields.Many2one('hr.employee', string="Employee's Name", readonly=True)
+    date_confirm = fields.Date(string='Confirmation Date', readonly=True)
+    date_valid = fields.Date(string='Validation Date', readonly=True)
+    department_id = fields.Many2one('hr.department', string='Department', readonly=True)
+    company_id = fields.Many2one('res.company', string='Company', readonly=True)
+    user_id = fields.Many2one('res.users', string='Validation User', readonly=True)
+    currency_id = fields.Many2one('res.currency', string='Currency', readonly=True)
+    price_total = fields.Float(string='Total Price', readonly=True, digits=dp.get_precision('Account'))
```

Comment:
Remove the decimal precision 'Account' and instead of it,  
- round by using the currency (foreign currency involved or company currency).
- store the full length of the decimal in db.